doc/specs/#658 - WinGet Download.md

```diff
@@ -0,0 +1,209 @@
+
+The following command alias are available
+
+The following arguments are available:
+
+The following options are available
+    --id                            Filter results by id 
+    --name                          Filter results by name
+    --moniker                       Filter results by moniker
+    --scope                         Select install scope (user or machine)
+    --locale                        Locale to use (BCP47 format)
+    --filename                      The value to rename the downloaded executable file to
+    -v, --version                   Use the specified version; default is the latest version
+    -s, --source                    Find package using the specified source
+    -a, --architecture              Select the architecture to install
+    -e, --exact                     Find package using exact match
+    -o, --output                    Location to download file(s) to
+    --installer-type                Specifies the filetype for the installer
+    --ignore-security-hash          Ignore the installer hash check failure
+    --accept-package-agreements     Accept all license agreements for packages
+    --help                          Shows help about the selected command
+    --wait                          |Prompts the user to press any key before existing
+    --verbose, --verbose-logs       Enables verbose logging for WinGet
+
+```
+
+WinGet search will be used to enlighten customers of the package's installers downloadable status. the new column will only be displayed if the application has an installer can be downloaded.
+
+```
+PS C:\> winget search Microsoft.VisualStudioCode
+Name                                  Id                                  Version Source Download
+-------------------------------------------------------------------------------------------------
+Microsoft Visual Studio Code          Microsoft.VisualStudioCode          1.75.0  winget True
+Microsoft Visual Studio Code Insiders Microsoft.VisualStudioCode.Insiders 1.76.0  winget
+```
+
+### WinGet Setting - Default Download Output
+
+A new setting will be included as the default output location for the user when package's installer is downloaded.
+
+```PS
+PS C:\> winget settings export
+{"$schema":"https://aka.ms/winget-settings-export.schema.json","adminSettings":{"BypassCertificatePinningForMicrosoftStore":false,"InstallerHashOverride":false,"LocalArchiveMalwareScanOverride":false,"LocalManifestFiles":false},"userSettingsFile":"C:\\Users\\Roy\\AppData\\Local\\Packages\\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\\LocalState\\settings.json","userDefaultAppOutput:"C:\\Users\\Roy\\Downloads\\"}
+```
+
+The following items will be included in the WinGet Settings Export Schema 
+
+```PS
+"network: {
+        "userDefaultAppOutput":"C:\\Users\\Roy\\Downloads\\"
```

Comment:
I think it should be in its own separate settings, not under "network" settings

useMeteredNetwork: DoDownloader honors the metered network settings, we probably don't want a separate settings for winget only, or at least it will be p2.